packages/patterns-react/src/patterns/SimpleLongForm/JumpLink.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings } from 'carbon-components';
+import { XAxis16 } from '@carbon/icons-react';
+const { prefix } = settings;
+
+/**
+ * Jump Link Component
+ *
+ * @param {object} props props object
+ * @param {string} props.href jump link link
+ * @param {string} props.text jump link text
+ * @returns {*} JumpLink component
+ */
+const JumpLink = ({ link: { href, text } }) => {
```

Comment:
Should this be an abstract pattern/component?

cc: @wonilsuhibm @oliviaflory @larahanlon2 @photodow @kennylam @annawen1 @andysherman2121

Reply:
Could we add this variation to the link+icon component we put in earlier this week?

if href starts with `#`
`↳ Link`
else if icon
`Link +`
else
`Link`

Reply:
I think we sould as we want to abstract components and solidify features